src/pages/AddItem/AddItem.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import ImageUpload from "../../components/ImageUpload/ImageUpload";
+import ItemNameInput from "../../components/ItemNameInput/ItemNameInput";
+import ItemDescription from "../../components/ItemDescription/ItemDescription";
+import PriceInput from "../../components/PriceInput/PriceInput";
+import TagInput from "../../components/TagInput/TagInput";
+import styles from "./AddItem.module.css";
+
+const AddItem = () => {
+  const [image, setImage] = useState(null);
+  const [description, setDescription] = useState("");
+  const [title, setTitle] = useState("");
+  const [price, setPrice] = useState("");
+  const [tags, setTags] = useState([]);
+
+  const isButtonDisabled =
+    !title.trim() ||
+    !description.trim() ||
+    !price.trim() ||
+    !image ||
+    !tags.length;
+
+  return (
+    <div className={styles.AddItem}>
+      <div className={styles.AddItemBar}>
+        <h2 className={styles.title}>상품 등록하기</h2>
+        <button
+          type="button"
```

Comment:
❗️ 수정요청
등록 버튼은 영역을 제출하는 것이므로 submit type이 적절합니다. 
```suggestion
          type="submit"
```